sdk/core/azure-core/test/ut/datetime.cpp

```diff
@@ -73,6 +73,59 @@ TEST(DateTime, ParseTimeRoundrip2)
   TestDateTimeRoundtrip("2013-11-19T14:30:59.1234567999Z", "2013-11-19T14:30:59.1234567Z");
 }
 
+TEST(DateTime, decimals)
+{
+  {
+    std::string strExpected("2020-10-13T21:06:15.3300000Z");
+    auto dt = DateTime::FromString("2020-10-13T21:06:15.33Z", DateTime::DateFormat::Iso8601);
+    auto const str2 = dt.ToIso8601String(DateTime::TimeFractionFormat::AllDigits);
```

Comment:
I don't understand this `AllDigits` enum value. We seem to be arbitrarily introducing a level of precision that the original data did not have. Do we need to support this particular format?

Reply:
All it needs it "please do not remove any zeros from the decimal part, even if the decimal is 0, just give the full digits, I need it"